kqueen/auth/common.py

```diff
@@ -9,21 +9,66 @@
 import importlib
 import logging
 
+config = current_config()
 logger = logging.getLogger('kqueen_api')
 
+"""
+Authentication Modules
+
+To define new module, need to specify it as dictionary, where:
+
+"auth_option_lower_case": {"engine": "EqualsToAuthClassName",
```

Comment:
Just a detail, but fix the indentation pls

Reply:
done